dapps/marketplace/src/constants/Growth$FbtEnum.js

```diff
@@ -7,10 +7,27 @@ const Growth$FbtEnum = {
   'growth.mar2019.short_name': 'Mar 2019',
   'growth.apr2019.name': 'April 2019 Campaign',
   'growth.apr2019.short_name': 'Apr 2019',
+  'growth.may2019.name': 'May 2019 Campaign',
+  'growth.may2019.short_name': 'May 2019',
   'growth.profile.and.email.requirement': 'Verified email and Origin profile',
   'growth.two.attestations.requirement': 'Two profile verifications',
   'growth.profile.requirement': 'Profile name and photo',
-  'growth.email.requirement': 'Verified email'
+  'growth.email.requirement': 'Verified email',
+  'growth.purchase.tshirt.title': 'Purchase an Origin HODL T-shirt',
+  'growth.purchase.tshirt.details': 'Earn 25 OGN per t-shirt purchased.',
+  'growth.purchase.gc.title': 'Purchase a $25 Amazon gift card',
+  'growth.purchase.gc.details':
+    'Earn 50 OGN per gift card purchased. Must be redeemed on Amazon US site.',
+  'growth.purchase.charity.title': 'Donate $100 to the GiveDirectly charity',
+  'growth.purchase.charity.details':
+    'Donation will be made by Origin on your behalf.',
+  'growth.purchase.housing.title': 'Book a home rental',
+  'growth.purchase.housing.details':
+    'Booking and stay must happen in the month of May.',
+  'growth.purchase.experience.title': 'TBD',
+  'growth.purchase.experience.details': 'TBD',
+  'growth.purchase.art.title': 'Purchase the XX by Trevor Jones',
```

Comment:
need to update this?

Reply:
Yes, I'm waiting on final copies and listing Ids from Matt...